service/src/test/java/org/mskcc/smile/service/SampleServiceTest.java

```diff
@@ -199,7 +199,7 @@ public void testGetSampleMetadataHistoryByIgoId() throws Exception {
     }
```

Comment:
Test case that I would want to make sure is added:

existing sample has updates coming from IGO/LIMS and the updates also happen to include non-IGO fields. 

For this example let's say that we receive a message on the IGO_NEW_REQUEST channel (i.e., from IGO LIMS) containing updates for an existing sample on the field `baitSet` (IGO-approved field) and `investigatorSampleId` (non-IGO approved field).

Once the updates are persisted and we fetch the latest version of this updated sample, we should see that this sample's `baitSet` value matches the expected updated value and `investigatorSampleId` matches what we were seeing previously, before the update.